### PR TITLE
Broken csv parsing

### PR DESCRIPTION
**Pull request by g-vega-cl**
## Purpose  
Make it so accents are not shown as unknown characters when parsing CSVs

## Changes and Additions  		 
- `.gitignore` added .envrc to set up environment variables.
- `Sidebar.tsx` was moved to their own folder and added the `quoteString` function.
- `ISidebar.ts` was added to contain the interfaces for `Sidebar.tsx`
- `Sidebar.test.ts` tests `quoteString` function

## Open Project Tickets

-[CSV parsing is broken in some instances; messages received from clients are being parsed into the incorrect columns w/in CSV](https://trello.com/c/vg6quwJu/74-csv-parsing-is-broken-in-some-instances-messages-received-from-clients-are-being-parsed-into-the-incorrect-columns-w-in-csv)

## Tested
- Branch is up to date to Master Branch
- Ran `yarn lint` before MR
- Ran `yarn test` before MR
- Application runs without crashes (`yarn start`)

Proof of fix (MacOS)
Screen Shot 2021-05-25 at 13 54 02

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.envrc
```
Comment (rweald) [inline .gitignore:20]: Ohh nice catch. I forget that I have a global gitignore on my local computer that prevents envrc from ever being committed.

File: src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
+import { type } from "node:os";
```
Comment (rweald) [inline src/components/Sidebar/Sidebar.tsx:1]: Why the need for this import? I might be missing something in our setup, but I've never seen the need to import `type` explicitly before.

**Review by rweald — COMMENTED** [review]
1 small question but overall 👍

File: src/components/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (g-vega-cl) [reply src/components/Sidebar/Sidebar.tsx]: My bad, that's an unused import. I tested the code without it and it works.

**Review by rweald — APPROVED** [review]

**Pull request merged**